pkg/action/executor/rescheduler.go

```diff
@@ -214,12 +214,12 @@ func (r *ReScheduler) reSchedule(action *turboaction.TurboAction) (*turboaction.
 	}
 
 	preScheduler, err := f(r.kubeClient, namespace, parentName, "", DefaultNoneExistSchedulerName)
+	defer f(r.kubeClient, namespace, parentName, DefaultNoneExistSchedulerName, preScheduler)
```

Comment:
yea, 
(1) If there is an  error, and fails to update the schedulerName, then preScheduler will be set to empty(""); in this case, the following call of f() in line-217 will do nothing.

(2) If there is an error, and the schedulerName  **may** have been updated, then the following call of f() in line-217 will try to restore the schedulerName.